who.py

```diff
@@ -273,14 +274,32 @@ def process_date(row):
     )
 
     isolower = countryiso.lower()
-    showcase = Showcase(
-        {
-            "name": f"{slugified_name}-showcase",
-            "title": f"Indicators for {countryname}",
-            "notes": f"Health indicators for {countryname}",
-            "url": f"http://www.who.int/countries/{isolower}/en/",
-            "image_url": f"http://www.who.int/sysmedia/images/countries/{isolower}.gif",
-        }
+
+    showcase = get_showcase(
+        retriever,
+        isolower,
+        countryname,
+        slugified_name,
+        alltags,
     )
-    showcase.add_tags(alltags)
     return dataset, showcase, bites_disabled
+
+
+def get_showcase(retriever, country_iso3, country_name, slugified_name, alltags):
+    try:
+        lower_iso3 = country_iso3.lower()
+        url = f"https://www.who.int/countries/{lower_iso3}/en/"
+        retriever.download_file(url)
+        showcase = Showcase(
+            {
+                "name": f"{slugified_name}-showcase",
+                "title": f"Indicators for {country_name}",
+                "notes": f"Health indicators for {country_name}",
+                "url": url,
+                "image_url": f"https://www.who.int/sysmedia/images/countries/{lower_iso3}.gif",
```

Comment:
Should this be https://cdn.who.int/media/images/default-source/countries-overview/flags/{lower_iso3}.jpg?

Reply:
Oops yes thanks!